src/DynamoCoreWpf/ViewModels/Core/InPortViewModel.cs

```diff
@@ -205,14 +204,27 @@ private void PortPropertyChanged(object sender, System.ComponentModel.PropertyCh
                     break;
                 case nameof(KeepListStructure):
                     RaisePropertyChanged(nameof(ShouldKeepListStructure));
-                    RefreshPortColors();
                     break;
                 case nameof(IsConnected):
-                    RefreshPortColors();
```

Comment:
What happened to this old function?

Reply:
This method is also being called in its parent `PortViewModel`. That's why this is not necessary anymore.